x-pack/plugins/security_solution/public/common/components/landing_page/onboarding/data_ingestion_hub_header/index.test.tsx

```diff
@@ -74,4 +100,14 @@ describe('WelcomeHeaderComponent', () => {
     const image = queryByTestId('data-ingestion-hub-header-image');
     expect(image).toHaveStyle({ backgroundImage: `url(${darkRocket})` });
   });
+
+  it('should display the modal when the "video" card is clicked', () => {
```

Comment:
You've mocked the mockUseDataIngestionHubHeaderCards to return no cards, so the watch video card should not exist. Could try removing the mock of useHeaderCards, and mock `useUsersUrl`.